src/widgets/EmbedWidget.ts

```diff
@@ -89,6 +89,8 @@ export default class EmbedWidget extends Widget {
       if (this._shouldFireLoadEvent()) {
         this._loadEvent(_sqh);
         _log("loaded");
+      } else {
```

Comment:
```suggestion
      } else if(frameDoc) {
```
can probably add this just to be safe